src/components/CardModal/CardModalActions/TagsMenu/CreateTagMenu/CreateTagMenu.js

```diff
@@ -50,6 +53,17 @@ const CreateTagMenu = () => {
     dispatch(showTagsMenu());
   };
 
+  const handleDeleteTag = () => {
+    dispatch(deleteTag(tagData.id));
+    dispatch(removeTagFromAllCards(tagData.id));
+
+    let updatedTags = [...currentSelectedCard.tags]
+    const tagIndex = updatedTags.indexOf(tagData.tagId);
+    updatedTags.splice(tagIndex, 1);
+    dispatch(setCurrentSelectedCard({ ...currentSelectedCard, tags: updatedTags }));
```

Comment:
currentSelectCard also needs to be updated so that the change is reflected in the modal